src/main/java/helper/LogInHelper.java

```diff
@@ -0,0 +1,22 @@
+package main.java.helper;
+
+import main.java.pages.header.UnlogUserInfo;
+import main.java.pages.user.UserPage;
+
+public class LogInHelper {
+
+    private UserPage userPage;
+
+    public LogInHelper () {
+        init();
+    }
+
+    private void init () {
+        userPage = new UnlogUserInfo().clickSignInHeaderFull().getLoginForm ()
+                .logIn ( "admin@gmail.com", "admin" );
+    }
+
+    public UserPage getUserPage() {
+        return userPage;
+    }
+}
```

Comment:
Nata already done this part and i corrected, so it is unnecessary to add this class

Reply:
deleted mu class